### PR TITLE
chore(readme): update readme

### PR DESCRIPTION
**Pull request by Eirsteir**
## Proposed changes
Gave readme a facelift and included some more general sections mainly directed at people not in index, since members of index will have access to our internal knowledge base/wiki.

Have a look at readme in the [branch](https://github.com/tihlde/Lepton/tree/chore/update-readme)  to see the changes

Issue number: No issue

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [x] Build (`make PR`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,156 +1,103 @@
...
+<br/>
+<p align="center">
+    <img width="80%" src="https://i.ibb.co/CtHxCph/Skjermbilde-2020-10-31-224329.png" alt="TIHLDEs nettside">   
+</p>
+<br/>
+
+
+## 🚀 Getting started
+
+Lepton requires python 3.6 or higher, docker and docker-compose. Other services are handled by docker.
```
Comment (norgor) [inline README.md:38]: The names should be changed here.
python -> Python
docker -> Docker
docker-compose -> Docker Compose
As these are their actual names.

File: README.md [hunk 2]
```diff
@@ -1,156 +1,103 @@
...
 ```
 
-#### Required Enviornment variables
-For the API to run you have to configure some enviornment variables. These variables can for example be put in a
-_.env_ file.
-First:
+From now on it's enough to run `make start` to run the application.
+
+We use [GNU Make](https://www.gnu.org/software/make/) to simplify common commands.
+Have a look at `makefile` to find out more about how to run the project without it.
```
Comment (norgor) [inline README.md:55]: should be the `makefile`

**Review by norgor — COMMENTED** [review]
Love the new look! Nice and short. 😄

**Review by Her0elt — APPROVED** [review]
Looks good

**Pull request merged**